src/site/content/en/handbook/redirects/index.md

```diff
@@ -10,26 +10,26 @@ This guide explains how to redirect content.
 
 ## Redirect a single page
 
-1. Open [`src/site/content/_redirects.yaml`][source].
+1. Open [`redirects.yaml`][source].
 1. Add an entry similar to the following:
 
    ```yaml
-   from: /path/to/old/page
-   to: /path/to/new/page
+   source: /path/to/old/page
+   destionation: /path/to/new/page
    ```
 
 ## Redirect multiple pages in a directory
 
 It's possible to redirect a subdirectory of content, but this feature
 is probably no longer useful because web.dev now uses a [flat URL structure][flat].
 
-1. Open [`src/site/content/_redirects.yaml`][source].
+1. Open [`redirects.yaml`][source].
 1. Add an entry similar to the following:
 
    ```yaml
-   from: /path/to/old/subdirectory/...
-   to: /path/to/new/subdirectory/...
+   source: /path/to/old/subdirectory/:patch
+   destionation: /path/to/new/subdirectory/:patch
```

Comment:
```suggestion
   destination: /path/to/new/subdirectory/:patch
```

Reply:
also if you rename `:patch` to `:path` or `:part` you'll want to update it here as well